admin/templates/preprints/preprint.html

```diff
@@ -104,6 +100,7 @@ <h2>Preprint: <b>{{ preprint.title }}</b> <a href="{{ preprint.absolute_url }}">
                         </tr>
                     {% endif %}
                     {% include "preprints/provider.html" with preprint=preprint %}
+                    {% include "preprints/machine_state.html" with preprint=preprint %}
```

Comment:
All you're comments boil down to missing this line, here's a screen shot showing the placement of the select box, the machine_state.html is included on the detail page with an option select (that disables if private or wrong transition) instead of a single button, but it will always be labeled.
![Screenshot 2024-07-19 at 6 29 25 AM](https://github.com/user-attachments/assets/2ce1f515-33f5-49af-b64a-014d155735ce)

Reply:
It appears collapsed on page load:
![Screenshot 2024-07-19 at 6 36 22 AM](https://github.com/user-attachments/assets/7f8c7e11-d89c-4a0b-b71b-4ec1a3b14745)